modules/runner/src/main/java/org/apache/ignite/internal/app/IgniteImpl.java

```diff
@@ -572,7 +573,8 @@ public class IgniteImpl implements Ignite {
                 new TransactionIdGenerator(() -> clusterSvc.nodeName().hashCode()),
                 () -> clusterSvc.topologyService().localMember().id(),
                 placementDriverMgr.placementDriver(),
-                partitionIdleSafeTimePropagationPeriodMsSupplier
+                partitionIdleSafeTimePropagationPeriodMsSupplier,
+                new SimpleFailHandler()
```

Comment:
We don't even know who will actually handle unrecoverable finish exceptions: tx coordinator, commit partition or someone else? We(at least I) also don't know how it'll be propagated to classes, e.g. it might be some static helper class. For know it's better just to log the exception without pretending that we will somehow handle it fail handler. In any case it should be covered within separate ticket.